src/main/java/org/arl/fjage/param/Parameter.java

```diff
@@ -22,4 +22,11 @@ public interface Parameter {
    */
   public int ordinal();
 
+  /**
+   * Gets the parameter name.
+   *
+   * @return unqualified parameter name
+   */
+  public String name();
```

Comment:
Doesn't this require updating every single `Parameter` implementation in fjage, UnetStack, etc? Sounds like a lot of work (and pain). Alternatively, we could change the definition of `toString()` to say that `toString()` has to be the unqualified parameter name. This is trivially true for all enum parameters, and then we can put the "remove `XXX.`" logic into `NamedParameter.toString()`.

Reply:
No change needed. All parameters are enums and all enums define `name()`. Adding it to the `Parameter` interface simply exposes it without having to cast.